components/CDPUserDashboard.tsx

```diff
@@ -40,6 +47,43 @@ const CDPZip = (props: CDPZipProps): JSX.Element => (
   </ListGroupItem>
 )
 
+/** Component which renders a a list of CDP files */
```

Comment:
Typo double "a".

Reply:
Thanks for catching! Fixed in 70e3ae96d53443799f08fcb41716e79c2e07151f